airbyte-integrations/connectors/destination-clickhouse/src/test/java/io/airbyte/integrations/destination/clickhouse/ClickhouseDestinationTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.clickhouse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.db.Databases;
+import io.airbyte.db.jdbc.JdbcDatabase;
+import io.airbyte.db.jdbc.JdbcUtils;
+import io.airbyte.integrations.base.AirbyteMessageConsumer;
+import io.airbyte.integrations.base.Destination;
+import io.airbyte.integrations.destination.ExtendedNameTransformer;
+import io.airbyte.protocol.models.AirbyteMessage;
+import io.airbyte.protocol.models.AirbyteMessage.Type;
+import io.airbyte.protocol.models.AirbyteRecordMessage;
+import io.airbyte.protocol.models.AirbyteStateMessage;
+import io.airbyte.protocol.models.CatalogHelpers;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.Field;
+import io.airbyte.protocol.models.JsonSchemaPrimitive;
+import java.time.Instant;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.testcontainers.containers.ClickHouseContainer;
+
+public class ClickhouseDestinationTest {
+
+  private static final String DB_NAME = "default";
+  private static final String STREAM_NAME = "id_and_name";
+  private static final ExtendedNameTransformer namingResolver = new ExtendedNameTransformer();
+
+  private static ClickHouseContainer db;
+  private static ConfiguredAirbyteCatalog catalog;
+  private static JsonNode config;
+
+  @BeforeAll
+  static void init() {
+    db = new ClickHouseContainer("yandex/clickhouse-server");
+    db.start();
+  }
+
+  @BeforeEach
+  void setup() {
+    catalog = new ConfiguredAirbyteCatalog().withStreams(List.of(
+        CatalogHelpers.createConfiguredAirbyteStream(
+            STREAM_NAME,
+            DB_NAME,
+            Field.of("id", JsonSchemaPrimitive.NUMBER),
+            Field.of("name", JsonSchemaPrimitive.STRING))));
+
+    config = Jsons.jsonNode(ImmutableMap.builder()
+        .put("host", db.getHost())
+        .put("port", db.getFirstMappedPort())
+        .put("database", DB_NAME)
+        .put("username", db.getUsername())
+        .put("password", db.getPassword())
+        .put("schema", DB_NAME)
+        .build());
+  }
+
+  @AfterAll
+  static void cleanUp() {
+    db.stop();
+    db.close();
+  }
+
+  // @Test
```

Comment:
Do not clear, why do we have this line commented? Whether add a comment or uncomment it, please.

Reply:
sorry forgot removing it, done.